sdk/highlight-py/pyproject.toml

```diff
@@ -50,8 +52,10 @@ pytest-cov = ">=4"
 pytest-asyncio = ">=0"
 loguru = ">=0"
 stripe = ">=6"
+redis = "^5.0.1"
```

Comment:
could we add a note to the e2e readme on how to run the fastapi / celery example and how we need to run redis for this to work?